src/services/packageService.ts

```diff
@@ -69,30 +88,80 @@ export class PackageService extends BaseService {
    * Cleans up generated function.json files after packaging has completed
    */
   public cleanUp() {
+    const functionFilesToRemove = [
+      "function.json",
+      "__init__.py"
+    ];
+
+    const functionFoldersToRemove = [
+      "__pycache__"
+    ]
+
+    const rootFilesToRemove = [
+      ".funcignore"
+    ]
+
     this.serverless.service.getAllFunctions().map((functionName) => {
       // Delete function.json if exists in function folder
-      const filePath = path.join(functionName, "function.json");
-      if (fs.existsSync(filePath)) {
-        fs.unlinkSync(filePath);
-      }
 
+      functionFilesToRemove.forEach((fileToRemove) => {
+        const filePath = path.join(functionName, fileToRemove);
+        if (fs.existsSync(filePath)) {
+          fs.unlinkSync(filePath);
+        }
+      });
+
+      functionFoldersToRemove.forEach((folderToRemove) => {
+        const folderPath = path.join(functionName, folderToRemove);
+        if (fs.existsSync(folderPath)) {
+          rimraf.sync(folderPath);
+        }
+      });
+      
       // Delete function folder if empty
       const items = fs.readdirSync(functionName);
       if (items.length === 0) {
         fs.rmdirSync(functionName);
       }
     });
 
+    for (const file of rootFilesToRemove) {
+      const filePath = path.join(this.serverless.config.servicePath, file);
+      if (fs.existsSync(filePath)) {
+        fs.unlinkSync(filePath);
+      }
+    }    
+
     return Promise.resolve();
   }
 
   /**
    * Creates the function.json for for the specified function
    */
   public createBinding(functionName: string, functionMetadata: FunctionMetadata) {
-    const functionJSON = functionMetadata.params.functionsJson;
-    functionJSON.entryPoint = functionMetadata.entryPoint;
-    functionJSON.scriptFile = functionMetadata.handlerPath;
+    const functionJSON = this.getFunctionJson(functionName, functionMetadata);
+    const functionDirPath = this.makeFunctionDir(functionName);
+
+    fs.writeFileSync(path.join(functionDirPath, "function.json"), this.stringify(functionJSON));
+
+    if (this.runtime.language === SupportedRuntimeLanguage.PYTHON) {
```

Comment:
What is happening here?  Writing an empty `__init__.py` file?  How does this work?  Would be good to have some comments here with intention of this change.

Reply:
I'll see if it works without it, but I've always had to create `__init__.py` in order for the directory to be considered a package. [Helpful article for more info](https://medium.com/@ramrajchandradevan/python-init-py-modular-imports-81b746e58aae#targetText=The%20__init__.py,on%20the%20module%20search%20path.)

Reply:
Removed this